exercises/bob/.meta/generator/bob_case.rb

```diff
@@ -2,8 +2,9 @@
 
 class BobCase < Generator::ExerciseCase
   def workload
-    indent_lines(["remark = #{input.inspect}",
-      "assert_equal '#{expected}', Bob.hey(remark), %q{Bob hears #{input.inspect}, and..}"
+    indent_lines(["remark = #{input["heyBob"].inspect}",
+      "assert_equal #{expected.inspect}, Bob.hey(remark), %q{Bob hears #{input["heyBob"].inspect}, and..}"
```

Comment:
💯 for using `inspect` to automatically handle the quoting correctly on the expected value.